config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+  get 'posts/index'
+  get 'users/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
Resource routing allows you to quickly declare all of the common routes for a given resourceful controller. A single call to resources can declare all of the necessary routes for your index , show , new , edit , create , update , and destroy actions.

- [ ] Use the `resources` for the configuration in the routes like that:

```
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thank you so much for this tip. I will definitely work on it.